release-notes.md

```diff
@@ -1,4 +1,5 @@
 ## 9.0.1
+* Bug 132060 Added migration to remove OpeningDate field from projects
```

Comment:
Cheers for doing these Dan, could you follow the convention on the other repos/this repo?

'Fixed bug XXXXX: blah blah'
'User Story XXXXX: Blah blah' etc etc

Also this isn't super clear as to the implications of this - i.e., Why has it been removed? in favour of what? We share these with Service Support to help them understand changes.

Reply:
I only think the context is extra important on this one as it is a column change that will get queried by the Service support team.